### ISSUE TITLE
Unable to Reproduce 3D Fine-tuning Experimental Results from Paper

### ISSUE THREAD

**Issue by fogradio** [HaoZhongkai/DPOT#8]
Hi,

I'm trying to run finetune3d.py to reproduce the DPOT-FT experimental results on 3D-Dataset PDB from the paper. However, my final L2RE value is 0.6358, which is significantly different from the 0.262 metric reported in the paper. Could you explain what might be causing this? Could the authors please provide detailed hyperparameter configurations?

My hyperparameter settings are as follows:
--model DPOT
--dataset ns2d
--train_path ns3d_pdb_M1_rand
--test_path ns3d_pdb_M1_rand
--resume_path /root/finetune_checkpoints/model_H.pth
--ntrain 90
--data_weights [1]
--use_writer True
--res 64
--noise_scale 0.0
--width 2048
--n_layers 27
--act gelu
--max_nodes -1
--modes 256
--use_ln 0
--normalize 0
--patch_size 8
--n_blocks 8
--mlp_ratio 4
--batch_size 4
--epochs 500
--lr 0.001
--opt adam
--beta1 0.9
--beta2 0.9
--lr_method step
--grad_clip 10000.0
--step_size 100
--step_gamma 0.5
--warmup_epochs 50
--sub 1
--T_in 10
--T_ar 1
--T_bundle 1
--gpu 0
--comment ""
--log_path /root/autodl-tmp/logs
--n_channels 4
--n_class 12
--load_components ['blocks']

**Comment by fogradio** [comment]
I’ve resolved the issue. The results reported in the paper were most likely obtained on the “ns3d_pdb_M1_turb” dataset. When I trained on this dataset using the hyperparameters mentioned above, the L2 relative error reached 0.23817, which is very close to the value reported in the original paper.
